app/src/main/java/io/appium/uiautomator2/server/mjpeg/MjpegScreenshotClient.java

```diff
@@ -30,14 +30,23 @@
 import static java.nio.charset.StandardCharsets.UTF_8;
 
 class MjpegScreenshotClient {
-    private static final byte[] START = 
-        "HTTP/1.0 200 OK\r\nServer: AQI Android Screenshot Socket Server\r\nConnection: close\r\nMax-Age: 0\r\nExpires: 0\r\nCache-Control: no-cache, private\r\nPragma: no-cache\r\nContent-Type: multipart/x-mixed-replace; boundary=--BoundaryString\r\n\r\n".getBytes(UTF_8);
+    private static final int INPUT_NOT_READY_SLEEP_TIME_MS = 100;
+    private static final byte[] START = (
+        "HTTP/1.0 200 OK\r\n" +
+        "Server: AQI Android Screenshot Socket Server\r\n" +
```

Comment:
what is AQI?

Reply:
Looks like that's a hold-over from our org name Applause App Quality, Inc. (AKA "AQI") -- I'll remove.